spec/features/users_can_sign_in_and_out_spec.rb

```diff
@@ -13,7 +13,19 @@
     expect(page).to have_content 'Sign out'
   end
 
-  scenario 'Signing out' do
+  scenario 'Signing in unsuccessfully' do
+    mock_sso_failure(:invalid_credentials)
+    mock_tasks_endpoint!
```

Comment:
Does a spec for an unsuccessful sign in need to mock these two endpoints?

Reply:
No, I just love cutting & pasting apparently 😳